internal/controller/standalone_pgadmin/pod.go

```diff
@@ -245,5 +303,47 @@ done
 
 	wrapper := `monitor() {` + startScript + reloadScript + `}; export cluster_file="$1"; export -f monitor; exec -a "$0" bash -ceu monitor`
 
-	return []string{"bash", "-ceu", "--", wrapper, "pgadmin", fmt.Sprintf("%s/~postgres-operator/%s", configMountPath, settingsClusterMapKey)}
+	return []string{"bash", "-ceu", "--", wrapper, "pgadmin", fmt.Sprintf("%s/%s", configMountPath, clusterFilePath)}
+}
+
+// startupCommand returns an entrypoint that prepares the filesystem for pgAdmin.
+func startupCommand() []string {
+	// pgAdmin reads from the `/etc/pgadmin/config_system.py` file during startup
+	// after all other config files.
+	// - https://github.com/pgadmin-org/pgadmin4/blob/REL-7_7/docs/en_US/config_py.rst
+	//
+	// This command writes a script in `/etc/pgadmin/config_system.py` that reads from
+	// the `pgadmin-settings.json` file and the `ldap-bind-password` file (if it exists)
+	// and sets those variables globally. That way those values are available as pgAdmin
+	// configurations when pgAdmin starts.
+	//
+	// Note: All pgAdmin settings are uppercase with underscores, so ignore any keys/names
+	// that are not.
+	//
+	// Note: set pgAdmin's LDAP_BIND_PASSWORD setting from the via Secret last
```

Comment:
:thinking: Should this be
```suggestion
	// Note: set pgAdmin's LDAP_BIND_PASSWORD setting from the Secret last
```
or perhaps
```suggestion
	// Note: set pgAdmin's LDAP_BIND_PASSWORD setting via Secret last
```
?